torch/nn/quantized/modules/linear.py

```diff
@@ -47,11 +72,11 @@ def __getstate__(self):
                                ' See https://github.com/pytorch/pytorch/issues/24045.'
                                ' Please use state_dict or torch.jit serialization.')
         qweight, bias = self._weight_bias()
-        return qweight, bias, self.training
+        return qweight, bias, self.training, self.dtype
```

Comment:
Do we still need this warning? The github issue is closed now.

Reply:
Not sure what you're referring to, https://github.com/pytorch/pytorch/issues/24045 is not closed

Reply:
Sorry, was referring to this one https://github.com/pytorch/pytorch/pull/29232